setup.py

```diff
@@ -55,7 +55,7 @@ def main():
                 "pytest >= 3.0.0", "pytest-cov", "pytest-mock", "pytest-timeout", "pytest-xdist"
             ],
             "docs": ["sphinx >= 1.6.3, < 2", "towncrier >= 17.8.0"],
-            "lint": ["flake8 == 3.5.0", "flake8-bugbear == 18.2.0", "pre-commit == 1.8.2"],
```

Comment:
probably want to add `flake8-bugbear==...` to the `additional_dependencies` of the `id: flake8` hook so that we continue to lint for these things

Reply:
also while doing that, probably need to limit `flake8` to `language_version: python3.6` since if I recall correctly `flake8-bugbear` only supports python3

Reply:
yeah, ammended